### PR TITLE
Update GBC 2023 Autumn wiki page (ZH+EN)

### PR DESCRIPTION
**Pull request by yobrevelc**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md [hunk 1]
```diff
@@ -33,27 +33,63 @@ The **GB Cup 2023 Autumn** (***GBC 2023 Autumn***) is a 3v3 captain draft–base
...
+| *2nd place* | 13$ |
+| *3rd place* | 10$ |
+| *4-8th place* | 8$ |
+| *9–16th place* | 6$ |
+| *All the other qualified players* | 2$ |
 
 ## Organisation
 
-The GB Cup 2023 Autumn is run by Team GB and various community members.
+The GB Cup 2022 Autumn was run by the Team GB and various community members.
```
Comment (untitledunrevised) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/en.md:84]:
```suggestion
The GB Cup 2023 Autumn was run by the Team GB and various community members.
```

File: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md [hunk 2]
```diff
@@ -33,27 +33,63 @@ The **GB Cup 2023 Autumn** (***GBC 2023 Autumn***) is a 3v3 captain draft–base
...
+| Every Winning Match in Regular Stage | 6$ |
+
+| Qualifier Placing | Prizes |
+| :-: | :-- |
+| *1st place* | 16$ |
+| *2nd place* | 13$ |
+| *3rd place* | 10$ |
+| *4-8th place* | 8$ |
+| *9–16th place* | 6$ |
+| *All the other qualified players* | 2$ |
```
Comment (untitledunrevised) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/en.md:80]:
```suggestion
| *1st place* | $16 |
| *2nd place* | $13 |
| *3rd place* | $10 |
| *4-8th place* | $8 |
| *9–16th place* | $6 |
| *All the other qualified players* | $2 |
```

File: wiki/Tournaments/GBC/GBC_2023_Autumn/zh.md [hunk 3]
```diff
@@ -33,16 +33,51 @@ tags:
...
 ## 奖励
 
-| 名次 | 奖励 |
+### QQ 赛区
+
+| 队伍排名 | 奖励 |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "冠军") | 个人资料页缎带，猫猫bot缎带, ¥150 |
+| ![Silver crown](/wiki/shared/crown-silver.png "亚军") | 猫猫bot缎带, ¥150 |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "季军") | 猫猫bot缎带, ¥120 |
```
Comment (untitledunrevised) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/zh.md:42]:
```suggestion
| ![Gold crown](/wiki/shared/crown-gold.png "冠军") | 个人资料页缎带，猫猫 bot 缎带，¥150 |
| ![Silver crown](/wiki/shared/crown-silver.png "亚军") | 猫猫 bot 缎带，¥150 |
| ![Bronze crown](/wiki/shared/crown-bronze.png "季军") | 猫猫 bot 缎带，¥120 |
```

File: wiki/Tournaments/GBC/GBC_2023_Autumn/zh.md [hunk 4]
```diff
@@ -33,16 +33,51 @@ tags:
...
-| *第 5-8 名* | *TBA* |
-| *第 9-16 名* | *TBA* |
-| *第 17-21 名* | *TBA* |
-| *第 22-24 名* | *TBA* |
+| *冠军* | ¥200 CNY |
+| *亚军* | ¥150 CNY |
+| *季军* | ¥100 CNY |
+| *第 7-8 名* | ¥75 CNY |
+| *第 9-24 名* | ¥50 CNY |
+| *其他所有入选的选手* | ¥20 CNY |
```
Comment (untitledunrevised) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/zh.md:57]:
```suggestion
| *冠军* | ¥200 |
| *亚军* | ¥150 |
| *季军* | ¥100 |
| *第 7-8 名* | ¥75 |
| *第 9-24 名* | ¥50 |
| *其他所有入选的选手* | ¥20 |
```

File: wiki/Tournaments/GBC/GBC_2023_Autumn/zh.md [hunk 5]
```diff
@@ -33,16 +33,51 @@ tags:
...
+
+### Discord 赛区
+
+| 队伍排名 | 奖励 |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "冠军") | Profile badge, ¥150 CNY |
+| ![Silver crown](/wiki/shared/crown-silver.png "亚军") | ¥150 CNY |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "季军") | ¥120 CNY |
```
Comment (untitledunrevised) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/zh.md:65]:
```suggestion
| ![Gold crown](/wiki/shared/crown-gold.png "冠军") | 个人资料页缎带，¥150 |
| ![Silver crown](/wiki/shared/crown-silver.png "亚军") | ¥150 |
| ![Bronze crown](/wiki/shared/crown-bronze.png "季军") | ¥120 |
```

File: wiki/Tournaments/GBC/GBC_2023_Autumn/zh.md [hunk 6]
```diff
@@ -33,16 +33,51 @@ tags:
...
+| *每场常规赛比赛胜利* | 6$ |
+
+| 资格赛排名 | 奖励 |
+| :-: | :-- |
+| *冠军* | 16$ |
+| *亚军* | 13$ |
+| *季军* | 10$ |
+| *第 4-8 名* | 8$ |
+| *第 9-16 名* | 6$ |
+| *其他所有入选的选手* | 2$ |
```
Comment (untitledunrevised) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/zh.md:80]:
```suggestion
| *每场常规赛比赛胜利* | $6 |

| 资格赛排名 | 奖励 |
| :-: | :-- |
| *冠军* | $16 |
| *亚军* | $13 |
| *季军* | $10 |
| *第 4-8 名* | $8 |
| *第 9-16 名* | $6 |
| *其他所有入选的选手* | $2 |
```

**Review by untitledunrevised — CHANGES_REQUESTED** [review]
纯细节上的排版问题修整

File: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md [hunk 7]
```diff
@@ -33,27 +33,63 @@ The **GB Cup 2023 Autumn** (***GBC 2023 Autumn***) is a 3v3 captain draft–base
...
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Profile badge, ¥150 CNY |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | ¥150 CNY |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | ¥120 CNY |
+| *4th place* | ¥90 CNY |
+| *5–6th place* | ¥75 CNY |
+| *7–8th place* | ¥60 CNY |
+| *9–12th place* | ¥45 CNY |
+| *13–16st place* | ¥30 CNY |
+| Every Winning Match in Regular Stage | 6$ |
```
Comment (untitledunrevised) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/en.md:71]:
```suggestion
| Every Winning Match in Regular Stage | $6 |
```

**Review by untitledunrevised — APPROVED** [review]

File: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md [hunk 8]
```diff
@@ -33,27 +33,63 @@ The **GB Cup 2023 Autumn** (***GBC 2023 Autumn***) is a 3v3 captain draft–base
...
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Profile badge, KanonBot profile badge, ¥150 CNY |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | KanonBot profile badge, ¥150 CNY |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | KanonBot profile badge, ¥120 CNY |
+| *4th place* | ¥90 CNY |
+| *5–6th place* | ¥75 CNY |
+| *7–8th place* | ¥60 CNY |
+| *9–12th place* | ¥45 CNY |
+| *13–16st place* | ¥30 CNY |
+| Every Winning Match in Regular Stage | ¥50 CNY |
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/en.md:48]: what does "Every Winning Match in Regular Stage" this mean? it doesn't make sense rn
it should also be in sentence case

File: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md [hunk 9]
```diff
@@ -33,27 +33,63 @@ The **GB Cup 2023 Autumn** (***GBC 2023 Autumn***) is a 3v3 captain draft–base
...
+
+| Qualifier Placing | Prizes |
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/en.md:50]:
```suggestion
| Qualifier placing | Prizes |
```

File: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md [hunk 10]
```diff
@@ -33,27 +33,63 @@ The **GB Cup 2023 Autumn** (***GBC 2023 Autumn***) is a 3v3 captain draft–base
 
 ## Prizes
 
-| Placing | Prizes |
+### QQ Division
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/en.md:36]:
```suggestion
### QQ division
```

File: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md [hunk 11]
```diff
@@ -33,27 +33,63 @@ The **GB Cup 2023 Autumn** (***GBC 2023 Autumn***) is a 3v3 captain draft–base
...
-| *17–21st place* | *TBA* |
-| *22–24th place* | *TBA* |
+| *1st place* | ¥200 CNY |
+| *2nd place* | ¥150 CNY |
+| *3rd place* | ¥100 CNY |
+| *4-8th place* | ¥75 CNY |
+| *9–24th place* | ¥50 CNY |
+| *All the other qualified players* | ¥20 CNY |
+
+### Discord Division
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/en.md:59]:
```suggestion
### Discord division
```

File: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md [hunk 12]
```diff
@@ -33,27 +33,63 @@ The **GB Cup 2023 Autumn** (***GBC 2023 Autumn***) is a 3v3 captain draft–base
...
+
+| Team Placing | Prizes |
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/en.md:61]:
```suggestion
| Team placing | Prizes |
```

File: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md [hunk 13]
```diff
@@ -33,27 +33,63 @@ The **GB Cup 2023 Autumn** (***GBC 2023 Autumn***) is a 3v3 captain draft–base
...
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Profile badge, ¥150 CNY |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | ¥150 CNY |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | ¥120 CNY |
+| *4th place* | ¥90 CNY |
+| *5–6th place* | ¥75 CNY |
+| *7–8th place* | ¥60 CNY |
+| *9–12th place* | ¥45 CNY |
+| *13–16st place* | ¥30 CNY |
+| Every Winning Match in Regular Stage | $6 |
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/en.md:71]: same as above

File: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md [hunk 14]
```diff
@@ -33,27 +33,63 @@ The **GB Cup 2023 Autumn** (***GBC 2023 Autumn***) is a 3v3 captain draft–base
...
+
+| Qualifier Placing | Prizes |
+| :-: | :-- |
+| *1st place* | $16 |
+| *2nd place* | $13 |
+| *3rd place* | $10 |
+| *4-8th place* | $8 |
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/en.md:78]:
```suggestion
| *4–8th place* | $8 |
```

File: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md [hunk 15]
```diff
@@ -33,27 +33,63 @@ The **GB Cup 2023 Autumn** (***GBC 2023 Autumn***) is a 3v3 captain draft–base
...
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | ¥150 CNY |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | ¥120 CNY |
+| *4th place* | ¥90 CNY |
+| *5–6th place* | ¥75 CNY |
+| *7–8th place* | ¥60 CNY |
+| *9–12th place* | ¥45 CNY |
+| *13–16st place* | ¥30 CNY |
+| Every Winning Match in Regular Stage | $6 |
+
+| Qualifier Placing | Prizes |
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/en.md:73]:
```suggestion
| Qualifier placing | Prizes |
```

File: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md [hunk 16]
```diff
@@ -33,27 +33,63 @@ The **GB Cup 2023 Autumn** (***GBC 2023 Autumn***) is a 3v3 captain draft–base
...
+| *2nd place* | $13 |
+| *3rd place* | $10 |
+| *4-8th place* | $8 |
+| *9–16th place* | $6 |
+| *All the other qualified players* | $2 |
 
 ## Organisation
 
-The GB Cup 2023 Autumn is run by Team GB and various community members.
+The GB Cup 2023 Autumn was run by the Team GB and various community members.
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/en.md:84]: why is this in past tense now? isn't the tournament still ongoing?

File: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md [hunk 17]
```diff
@@ -33,27 +33,63 @@ The **GB Cup 2023 Autumn** (***GBC 2023 Autumn***) is a 3v3 captain draft–base
...
 
 | Position | Member(s) |
 | :-- | :-- |
 | Host | ::{ flag=CN }:: [\[GB\]yobrevelc](https://osu.ppy.sh/users/14128407) |
+| Staff | ::{ flag=KR }:: [\[GB\]SuddenDeath](https://osu.ppy.sh/users/6699923), ::{ flag=CN }:: [\[GB\]Mafufu](https://osu.ppy.sh/users/10884561), ::{ flag=CN }:: [\[GB\]GanyuAngel](https://osu.ppy.sh/users/27233308) |
 | Mappool selector | ::{ flag=CN }:: [\[GB\]yobrevelc](https://osu.ppy.sh/users/14128407), ::{ flag=CN }:: [\[GB\]Thaumiel](https://osu.ppy.sh/users/15017771), ::{ flag=CN }:: [\[GB\]gluefly](https://osu.ppy.sh/users/8715409), ::{ flag=CN }:: [\[GB\]DiGua](https://osu.ppy.sh/users/25308131) |
-| SV mappool selector | ::{ flag=CN }:: [\[GB\]Mafufu](https://osu.ppy.sh/users/10884561) |
-| Mappool helper | *TBA* |
+| SV Mappool selector | ::{ flag=CN }:: [\[GB\]Mafufu](https://osu.ppy.sh/users/10884561), ::{ flag=VE }:: [Lott](https://osu.ppy.sh/users/13821222) |
+| Custom Mapper | *TBA* |
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/en.md:92]:
```suggestion
| SV mappool selector | ::{ flag=CN }:: [\[GB\]Mafufu](https://osu.ppy.sh/users/10884561), ::{ flag=VE }:: [Lott](https://osu.ppy.sh/users/13821222) |
| Custom mapper | *TBA* |
```

File: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md [hunk 18]
```diff
@@ -71,6 +107,58 @@ The GB Cup 2023 Autumn is run by Team GB and various community members.
 - [GBC Discord server](https://discord.gg/kR4hJnPs)
 - [Information spreadsheet](https://docs.qq.com/sheet/DTVlRcExwbUpTVVZn?tab=BB08J2)
 
+## Participants
+
+### QQ Division
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/en.md:112]:
```suggestion
### QQ division
```

File: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md [hunk 19]
```diff
@@ -71,6 +107,58 @@ The GB Cup 2023 Autumn is run by Team GB and various community members.
...
+| Q22 | **和对面凑一队** | ::{ flag=CN }:: **[Wings_AnChen](https://osu.ppy.sh/users/12694894)**, ::{ flag=CN }:: [ATH_1gr](https://osu.ppy.sh/users/11931583), ::{ flag=CN }:: [xibuniuzai](https://osu.ppy.sh/users/10513110), ::{ flag=CN }:: [Apr4th](https://osu.ppy.sh/users/20485400), ::{ flag=CN }:: [neeeeeh](https://osu.ppy.sh/users/18586390) |
+| Q23 | **再奖励一把大天空** | ::{ flag=CN }:: **[Yurucawaii](https://osu.ppy.sh/users/25682415)**, ::{ flag=CN }:: [- Xiaoluoli -](https://osu.ppy.sh/users/9502281), ::{ flag=CN }:: [Tempest_WH](https://osu.ppy.sh/users/16039216), ::{ flag=CN }:: [CLLbin](https://osu.ppy.sh/users/11837219), ::{ flag=CN }:: [mayoi123](https://osu.ppy.sh/users/25379764) |
+| Q24 | **猫猫统治世界** | ::{ flag=CN }:: **[Carpihat](https://osu.ppy.sh/users/10085090)**, ::{ flag=CN }:: [Akermo](https://osu.ppy.sh/users/14934242), ::{ flag=CN }:: [\[GB\]Azukisan](https://osu.ppy.sh/users/5378805), ::{ flag=CN }:: [superSSS](https://osu.ppy.sh/users/4315477), ::{ flag=CN }:: [\[GB\] limata](https://osu.ppy.sh/users/21326517) |
+
+### Discord Division
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/en.md:141]:
```suggestion
### Discord division
```

File: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md [hunk 20]
```diff
@@ -123,7 +211,7 @@ The GB Cup 2023 Autumn is run by Team GB and various community members.
 ### Tournament registration
 
 1. Please send the host a **private message** and **fill [this form](https://wj.qq.com/s2/13179234/c366/)** to register. 
-   - Players need to choose a division according to their chat software, i.e. the Discord division or QQ division. Both the QQ division and Discord division will have an independent drafting phase and regular stage.
+   - Players need to choose a division according to their chat software, i.e. the Discord division or QQ division. Both the QQ division and Discord division will have an independent tournament, but use same mapppool.
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/en.md:214]:
```suggestion
   - Players need to choose a division according to their chat software, i.e. the Discord division or QQ division. The QQ division and Discord division will each have an independent tournament, but use same mappool.
```

File: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md [hunk 21]
```diff
@@ -157,7 +245,7 @@ The GB Cup 2023 Autumn is run by Team GB and various community members.
 
 1. Following the Qualifier stage, all players will participate in a phase to form teams, called the *drafting phase*. Both divisions will have independent drafting phases. All unpicked players will be eliminated from the tournament.
 2. Following the drafting phase, a Swiss stage will be played in both zones, called the *regular stage*. 16 teams will advance in total, and the others get eliminated from the tournament.
-3. Following the regular stage, the elimination stage will be played, called *playoffs*. The top 8 teams from the regular stage will advance to a winner bracket, other teams will get into loser bracket.
+3. Following the regular stage, the elimination stage will be played in both zones, called *playoffs*. The top 8 teams from the regular stage will advance to a winner bracket, other teams will get into loser bracket.
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/en.md:248]:
```suggestion
3. Following the regular stage, the elimination stage will be played in both divisions, called *playoffs*. The top 8 teams from the regular stage will advance to a winner bracket, other teams will get into loser bracket.
```

File: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md [hunk 22]
```diff
@@ -187,10 +275,10 @@ The GB Cup 2023 Autumn is run by Team GB and various community members.
 
 ### Regular phase instructions
 
-1. The regular phase will be divided into two division, the Asia division using Discord, and the China division using QQ.
+1. The regular phase will be divided into two division, the Discord division and the QQ division.
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/en.md:278]:
```suggestion
1. The regular phase will be divided into two divisions, the Discord division and the QQ division.
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md (hunk 8 above)
Comment (Walavouchey) [reply wiki/Tournaments/GBC/GBC_2023_Autumn/en.md:48]:
```suggestion
| *For every match won in the regular stage* | ¥50 CNY |
```

File: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md (hunk 13 above)
Comment (Walavouchey) [reply wiki/Tournaments/GBC/GBC_2023_Autumn/en.md:71]:
```suggestion
| *For every match won in the regular stage* | $6 |
```

**Comment by Walavouchey** [comment]
force-pushed to combine 40+ commits
in the future, apply reviews in batches as described in https://osu.ppy.sh/wiki/en/osu%21_wiki/Contribution_guide/Best_practices#applying-reviews

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**